app/code/Magento/Bundle/README.md

```diff
@@ -1,2 +1,126 @@
+    - `collection` is bundle collection of selection before count prices of it (`\Magento\Bundle\Model\ResourceModel\Selection\Collection` class).
+    - `store_id` is a bundle product store id.
+- `catalog_product_prepare_index_select` in the `\Magento\Bundle\Model\ResourceModel\Indexer\Price::prepareBundlePriceByType` method. Parameters:
+    - `select` is a select for bundle price by type to DB (`\Magento\Framework\DB\Select` class)
+    - `entity_field` is a entity id column (`\Zend_Db_Expr` class)
+    - `website_field` is a website id column (`\Zend_Db_Expr` class)
+    - `store_field` is a default store id column (`\Zend_Db_Expr` class)
+
+For information about an event in Magento 2, see [Events and observers](http://devdocs.magento.com/guides/v2.1/extension-dev-guide/events-and-observers.html#events).
+
+### Layouts
+
+This module introduces the following layouts and layout handles in the directories:
+- `view/adminhtml/layout`:
+    - `adminhtml_order_shipment_new`
+    - `adminhtml_order_shipment_view`
+    - `catalog_product_bundle`
+    - `catalog_product_new`
+    - `catalog_product_view_type_bundle`
+    - `customer_index_wishlist`
+    - `sales_order_creditmemo_new`
+    - `sales_order_creditmemo_updateqty`
+    - `sales_order_creditmemo_view`
+    - `sales_order_invoice_new`
+    - `sales_order_invoice_updateqty`
+    - `sales_order_invoice_view`
+    - `sales_order_invoice_view`
+- `view/base/layout`:
+  - `catalog_product_prices`
+- `view/frantend/layout`:
+    - `catalog_product_view_type_bundle`
+    - `catalog_product_view_type_simple`
+    - `checkout_cart_configure_type_bundle`
+    - `checkout_cart_item_renderers`
+    - `checkout_onepage_review_item_renderers`
+    - `default`
+    - `sales_email_order_creditmemo_renderers`
+    - `sales_email_order_invoice_renderers`
+    - `sales_email_order_renderers`
+    - `sales_email_order_shipment_renderers`
+    - `sales_order_creditmemo_renderers`
+    - `sales_order_invoice_renderers`
+    - `sales_order_item_renderers`
+    - `sales_order_print_creditmemo_renderers`
+    - `sales_order_print_invoice_renderers`
+    - `sales_order_print_renderers`
+    - `sales_order_print_shipment_renderers`
+    - `sales_order_shipment_renderers`
+
+For more information about a layout in Magento 2, see the [Layout documentation](http://devdocs.magento.com/guides/v2.1/frontend-dev-guide/layouts/layout-overview.html).
+
+### UI components
+
+You can extend a product and widgets updates using the configuration files located in the directories 
+- `view/adminhtml/ui_component`:
+    - 'bundle_product_listing'
+- `view/frontend/ui_component`:
+    - 'widget_recently_compared'
+    - 'widget_recently_viewed'
+
+For information about a UI component in Magento 2, see [Overview of UI components](http://devdocs.magento.com/guides/v2.1/ui_comp_guide/bk-ui_comps.html).
+
+### Public APIs
+
+`\Magento\Bundle\Api\ProductLinkManagementInterface`:
+
+- get all children for Bundle product.
```

Comment:
Please, consider using a lower-cased version of the "bundle" product everywhere (you use both versions)
```suggestion
- get all children for bundle product.
```